ansible/roles/ansible_service_broker_setup/tasks/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
+  - name: Waiting 1 minutes for service-catalog
+    action: >-
+      shell "{{ oc_cmd }}" get deployments -n service-catalog apiserver
```

Comment:
-n kube-service-catalog worked here for me.

Reply:
This patch in conjunction with these changes to the template gave me a successful run with coalmine and 3.7, though I did not see any APB's populated:
https://github.com/openshift/ansible-service-broker/pull/422

```
diff --git a/ansible/roles/ansible_service_broker_setup/tasks/main.yml b/ansible/roles/ansible_service_broker_setup/tasks/main.yml
index 335353d..2e712db 100644
--- a/ansible/roles/ansible_service_broker_setup/tasks/main.yml
+++ b/ansible/roles/ansible_service_broker_setup/tasks/main.yml
@@ -83,6 +83,14 @@
         -p SANDBOX_ROLE={{ broker_sandbox_role }}
     when: "not rcm"
 
+  - name: Update Broker Kind and Auth when using coalmine
+    set_fact:
+      cmd_process_asb_template: >-
+        {{ cmd_process_asb_template }}
+        -p BROKER_KIND="ServiceBroker"
+        -p BROKER_AUTH='{ "basic": { "secretRef": { "namespace": "ansible-service-broker", "name": "asb-auth-secret" } } }'
+    when: coalmine is defined and coalmine
+
   - name: Process template "{{ local_target_asb_template }}"
     shell: >
       {{ cmd_process_asb_template }}

```